### ISSUE TITLE
Move to NPM package to organization

### ISSUE THREAD

**Issue by chrisdmacrae** [forestryio/create-static-site#38]
Currently, the NPM package is published under @ncphillips name. This means we can't delegate publishing minor versions of the package. (E.g, see #35).

I create an organization called `create-static-site` and invite @ncphillips. I propose we move the package to either:
- the create-static-site organization
- or the forestryio organization

That way we can have multiple managers of the public package.

**Comment by ncphillips** [comment]
I don't think we need to move the project to do that, I just have to add collaborators to packages on NPM

**Comment by chrisdmacrae** [comment]
@ncphillips TIL!
